From 5355a17d2b3b9565f34be9b3f082515c767adbf3 Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Mon, 06 May 2019 11:17:36 +0300
Subject: [PATCH] [SW-0] pll: inline one-liner functions

hl_pll_reset() and hl_pll_ack_int() are one liners which are just
writes to a register.
Inline them both.

Change-Id: I20761736dfdbc7d20e609f25e4489fe59dd42caa
Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
---

diff --git a/common/pll.c b/common/pll.c
index c717f61..5438faf 100644
--- a/common/pll.c
+++ b/common/pll.c
@@ -31,7 +31,7 @@
 		;
 }
 
-static void hl_pll_reset(struct hl_pll_cfg *cfg, bool reset)
+static inline void hl_pll_reset(struct hl_pll_cfg *cfg, bool reset)
 {
 	regmap_writel(reset, cfg->base + HL_PLL_RST);
 }
@@ -133,7 +133,7 @@
 	}
 }
 
-static void hl_pll_ack_int(struct hl_pll_cfg *cfg)
+static inline void hl_pll_ack_int(struct hl_pll_cfg *cfg)
 {
 	regmap_writel(1, cfg->base + HL_PLL_LOCK_INTR);
 }
